tests/unit/test-firefox/test.firefox.js

```diff
@@ -35,6 +35,37 @@ function withBaseProfile(callback) {
   );
 }
 
+function createFakeProfileFinder(profilesDirPath) {
+  const FakeProfileFinder = sinon.spy((...args) => {
+    const finder = new FirefoxProfile.Finder(...args);
+
+    sinon.spy(finder, 'readProfiles');
+
+    return finder;
+  });
+
+  FakeProfileFinder.locateUserDirectory = sinon.spy(() => {
+    return profilesDirPath;
+  });
+
+  return FakeProfileFinder;
+}
+
+async function createFakeProfilesIni(dirPath, profilesDefs) { // eslint-disable-line
+  let content = '';
+
+  for (const [idx, profile] of profilesDefs.entries()) {
+    content += `[Profile${idx}]\n`;
+    for (const k of Object.keys(profile)) {
+      // $FLOW_FIXME: test123
```

Comment:
@kumar303 @saintsebastian it is an acronym :-) it means: "Damn... we will have a chat you and me... later... for now just shut up" :-P

Thank you both for spotting it, I'm going to have a chat with flow about this and get back to you with more info about it :-D